core/dist/css/decanter-grid.css

```diff
@@ -1,1910 +0,0 @@
-html {
```

Comment:
I'm pretty sure we want to keep this file. Run and commit a new build.

Reply:
Hm, seems like my stuff did more than I wanted it to. So yes.